### PR TITLE
QUALITY: remove @ajirving's email and replace with Github account.

### PR DESCRIPTION
**Pull request by njsch**
I'm pretty sure this email address no longer works and I don't know whether we have permission to use his Gmail or any other email, so this now re-directs to his Github in /readme.md.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ line of LaTeX at any one time, however when scrolling through a
 LaTeX document the braille is refreshed on the fly.  Currently, support for cursor routing buttons on Braille displays is only provided when working with Brltty under Linux.
 
 
-The scripts also translate the current line into English speech which
+The scripts also translate the current line into English speech wghich
```
Comment (lukaszgo1) [inline readme.md:18]: Please revert this change.

**Review by lukaszgo1 — CHANGES_REQUESTED** [review]
LGTM assuming the unnecessary change is reverted.

**Comment by njsch** [comment]
Closed upon request.

**Comment by lukaszgo1** [comment]
I'm afraid there is a misunderstanding here. I only asked you to restore the line 18 to its previous form.

**Comment by njsch** [comment]
Oh shoot. Yep. That letter `G`. Gotcha. I'll put it back in later and submit a pr.

**Comment by lukaszgo1** [comment]
> Oh shoot. Yep. That letter `G`. Gotcha. I'll put it back in later and submit a pr.


You can just restore this branch, push the change and reopen this one.

**Comment by njsch** [comment]
@lukaszgo1, try this.

**Review by lukaszgo1 — APPROVED** [review]

**Pull request merged**